apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -59,6 +72,7 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 
 	const domainSummary = (
 		<div className="nux-launch__summary-item">
+			{ /* TODO: what if the user picks a monthly-billed plan?  */ }
```

Comment:
See https://github.com/Automattic/wp-calypso/issues/48760#issuecomment-766952241

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/a4e89e697398c082479c44f948d5604d081cc501